src/client/common/contextKey.ts

```diff
@@ -0,0 +1,15 @@
+import { commands } from 'vscode';
+
+export class ContextKey {
+    private lastValue: boolean;
+
+    constructor(private name: string) { }
+
+    public async set(value: boolean): Promise<void> {
+        if (this.lastValue === value) {
+            return;
+        }
+        this.lastValue = value;
+        await commands.executeCommand('setContext', this.name, this.lastValue);
```

Comment:
Is `setContext` an Electron/Chrome thing?

Reply:
vscode thingy